tests/FSharp.Control.AsyncSeq.Tests/FSharp.Control.AsyncSeq.Tests.fsproj

```diff
@@ -9,8 +9,8 @@
     <OutputType>Library</OutputType>
     <RootNamespace>FSharp.Control.AsyncSeq.Tests</RootNamespace>
     <AssemblyName>FSharp.Control.AsyncSeq.Tests</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
-    <TargetFSharpCoreVersion>4.3.0.0</TargetFSharpCoreVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFSharpCoreVersion>4.4.0.0</TargetFSharpCoreVersion>
```

Comment:
We still need to target 4.3.0.0 to allow this library to be used by maximal number of users - I believe you should be able to continue to do that - though full stack traces will only be given when 4.4.0.0 is used at runtime.